tests/unit/amo/components/TestSearchContextCard.js

```diff
@@ -104,4 +109,125 @@ describe('SearchContextCard', () => {
     expect(root.find('.SearchContextCard-header'))
       .toIncludeText('No add-ons found');
   });
+
+  // addonType = ADDON_TYPE_THEME
+  it('should render singular form when only one result is found with addonType as ADDON_TYPE_THEME', () => {
+    const { store } = dispatchSearchResults({
+      addons: { [fakeAddon.slug]: fakeAddon },
+      filters: {
+        addonType: ADDON_TYPE_THEME,
+        query: 'test',
+      },
+    });
+
+    const root = render({ store });
+
+    expect(root.find('.SearchContextCard-header'))
+      .toIncludeText('1 theme found for "test"');
+  });
+
+  it('should render plural form when multiple results are found with addonType as ADDON_TYPE_THEME', () => {
+    const { store } = dispatchSearchResults({
+      filters: {
+        addonType: ADDON_TYPE_THEME,
+        query: 'test',
+      },
+    });
+
+    const root = render({ store });
+
+    expect(root.find('.SearchContextCard-header'))
+      .toIncludeText('2 themes found for "test"');
+  });
+
+  // addonType = ADDON_TYPE_DICT
+  it('should render singular form when only one result is found with addonType as ADDON_TYPE_DICT', () => {
+    const { store } = dispatchSearchResults({
+      addons: { [fakeAddon.slug]: fakeAddon },
+      filters: {
+        addonType: ADDON_TYPE_DICT,
+        query: 'test',
+      },
+    });
+
+    const root = render({ store });
+
+    expect(root.find('.SearchContextCard-header'))
+      .toIncludeText('1 dictionary found for "test"');
+  });
+
+  it('should render plural form when multipe results are found with addonType as ADDON_TYPE_DICT', () => {
+    const { store } = dispatchSearchResults({
+      filters: {
+        addonType: ADDON_TYPE_DICT,
+        query: 'test',
+      },
+    });
+
+    const root = render({ store });
+
+    expect(root.find('.SearchContextCard-header'))
+      .toIncludeText('2 dictionaries found for "test"');
+  });
+
+  // addonType = ADDON_TYPE_EXTENSION
+  it('should render singular form when only one result is found with addonType = ADDON_TYPE_EXTENSION', () => {
+    const { store } = dispatchSearchResults({
+      addons: { [fakeAddon.slug]: fakeAddon },
+      filters: {
+        addonType: ADDON_TYPE_EXTENSION,
+        query: 'test',
+      },
+    });
+
+    const root = render({ store });
+
+    expect(root.find('.SearchContextCard-header'))
+      .toIncludeText('1 extension found for "test"');
+  });
+
+  it('should render plural form when multipe results are found with addonType as ADDON_TYPE_EXTENSION', () => {
+    const { store } = dispatchSearchResults({
+      filters: {
+        addonType: ADDON_TYPE_EXTENSION,
+        query: 'test',
+      },
+    });
+
+    const root = render({ store });
+
+    expect(root.find('.SearchContextCard-header'))
+      .toIncludeText('2 extensions found for "test"');
+  });
+
+  // addonType = ADDON_TYPE_EXTENSION
```

Comment:
You don't need these comments here, you wrote really good test specs actually (👍🏻) and the `dispatchSearchResults` calls show which type you're using so you can remove these comments.